cmake/modules/hunter_dump_cmake_flags.cmake

```diff
@@ -20,6 +20,25 @@ function(hunter_dump_cmake_flags)
     hunter_internal_error("Unparsed arguments: ${x_UNPARSED_ARGUMENTS}")
   endif()
 
+
+  if(APPLE)
+    if(IOS)
+      string(COMPARE EQUAL "${IOS_DEPLOYMENT_SDK_VERSION}" "" _no_deployment_sdk_version)
+      if(_no_deployment_sdk_version)
+        set(CMAKE_CXX_FLAGS "-miphoneos-version-min=@IOS_SDK_VERSION@")
```

Comment:
This file is not configured so it should be `${IOS_SDK_VERSION}` instead of `@IOS_SDK_VERSION@`. Also please check that this string is not empty by `hunter_test_string_not_empty("${IOS_SDK_VERSION}")`.